flocker/node/agents/test/test_cinder.py

```diff
@@ -61,3 +67,65 @@ def test_verify_false_ca_path(self):
             'verify_ca_path': '/a/path'
         }
         self.assertEqual(_openstack_verify_from_config(**config), False)
+
+
+class FakeCinderClient(object):
```

Comment:
Also worth pointing out there's a project for building a test-friendly fake of the OpenStack API, https://github.com/rackerlabs/mimic

I think we're not using it because it doesn't do Cinder.  But maybe we should consider contributing...